src/BenchmarkTools.jl

```diff
@@ -46,6 +48,8 @@ export BenchmarkGroup,
        invariants,
        regressions,
        improvements,
+       median,
```

Comment:
Listed twice?

Reply:
I assumed a separate export was need to for both `::Trial` and `::BenchmarkGroup`, but with the magic of Julia, it seems like only one is necessary. Will update to list these only under Trial.